eAbsentee/form/templates/form.html

```diff
@@ -42,14 +42,14 @@
         <h1 class="text-capitalize text-center text-light"> Vote by Mail </h1>
         <h3 class="text-center text-light"> Apply for Your Absentee Ballot Today</h3>
         <div class="">
+          <br>
           <ul>
-            <br />
-            <p class="text-light font-weight-bold mb-1">• Effective July 1st, 2020, any registered Virginia voter can request an absentee mail-in ballot. No reason is required!</p>
-            <p class="text-light font-weight-bold mb-1">• Now is the best time to apply for your absentee mail-in ballot, in order to avoid the rush. Ballots for the November 3rd Election will be mailed out to voters beginning September 18th.</p>
-            <p class="text-light font-weight-bold mb-1">• The mail-in ballot application deadline is 5pm Friday, October 23rd. With unprecedented demand, apply now to reduce any possible delays.</p>
-            <p class="text-light font-weight-bold mb-1">• Return your completed mail-in ballot to the registrar as soon as possible. The last day to send it in is November 3rd and it must be receieved by November 6th.</p>
-            <p class="text-light font-weight-bold mb-1">• Does your voter record need to be updated for a change in your name or address? You can do so at the end of this absentee mail-in ballot application.</p>
-            <p class="text-light font-weight-bold mb-1">• If you are not registered to vote, register to vote <u><a href="https://www.elections.virginia.gov/citizen-portal/" target="_blank">here</a></u> first, then return to eAbsentee to complete your application.</p>
+           <li><p class="text-light font-weight-bold mb-1">Effective July 1st, 2020, any registered Virginia voter can request an absentee mail-in ballot. No reason is required!</p></li>
+            <li><p class="text-light font-weight-bold mb-1">Now is the best time to apply for your absentee mail-in ballot, in order to avoid the rush. Ballots for the November 3rd Election will be mailed out to voters beginning September 18th.</p></li>
+            <li><p class="text-light font-weight-bold mb-1">The mail-in ballot application deadline is 5pm Friday, October 23rd. With unprecedented demand, apply now to reduce any possible delays.</p></li>
+            <li><p class="text-light font-weight-bold mb-1">Return your completed mail-in ballot to the registrar as soon as possible. The last day to send it in is November 3rd and it must be receieved by November 6th.</p></li>
+            <li><p class="text-light font-weight-bold mb-1">Does your voter record need to be updated for a change in your name or address? You can do so at the end of this absentee mail-in ballot application.</p></li>
+           <li><p class="text-light font-weight-bold mb-1">If you are not registered to vote, register to vote <u><a href="https://www.elections.virginia.gov/citizen-portal/" target="_blank">here</a></u> first, then return to eAbsentee to complete your application.</p></li>
```

Comment:
(indentation)
```suggestion
            <li><p class="text-light font-weight-bold mb-1">Effective July 1st, 2020, any registered Virginia voter can request an absentee mail-in ballot. No reason is required!</p></li>
            <li><p class="text-light font-weight-bold mb-1">Now is the best time to apply for your absentee mail-in ballot, in order to avoid the rush. Ballots for the November 3rd Election will be mailed out to voters beginning September 18th.</p></li>
            <li><p class="text-light font-weight-bold mb-1">The mail-in ballot application deadline is 5pm Friday, October 23rd. With unprecedented demand, apply now to reduce any possible delays.</p></li>
            <li><p class="text-light font-weight-bold mb-1">Return your completed mail-in ballot to the registrar as soon as possible. The last day to send it in is November 3rd and it must be receieved by November 6th.</p></li>
            <li><p class="text-light font-weight-bold mb-1">Does your voter record need to be updated for a change in your name or address? You can do so at the end of this absentee mail-in ballot application.</p></li>
            <li><p class="text-light font-weight-bold mb-1">If you are not registered to vote, register to vote <u><a href="https://www.elections.virginia.gov/citizen-portal/" target="_blank">here</a></u> first, then return to eAbsentee to complete your application.</p></li>
```